pkg/kube/client.go

```diff
@@ -374,7 +374,14 @@ func deleteResource(c *Client, info *resource.Info) error {
 	if err != nil {
 		// If there is no reaper for this resources, delete it.
 		if kubectl.IsNoSuchReaperError(err) {
-			return resource.NewHelper(info.Client, info.Mapping).Delete(info.Namespace, info.Name)
+			propagationPolicy := metav1.DeletePropagationBackground
+			return resource.NewHelper(info.Client, info.Mapping).DeleteWithOptions(
+				info.Namespace,
+				info.Name,
+				&metav1.DeleteOptions{
+					PropagationPolicy: &propagationPolicy,
```

Comment:
@adamreese or @bacongobbler Do either of you know if this might have any secondary side effects? Essentially this would be telling it to cascade delete anything in the background, so I am not sure if that could introduce an error down the road

Reply:
Possibly. Looking 👀